### PR TITLE
Implements PriceQuote adapter

### PR DESCRIPTION
**Pull request by alxberardi**

### CODE REVIEW

**Comment by msmirnoff** [comment]
👍

File: lib/quick_travel/price_quote.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+module QuickTravel
+  class PriceQuote < Adapter
+    self.api_base = '/api/price_quotes'
+
+    def self.calculate(params)
+      response = post_and_validate("#{api_base}/calculate", params)
+      new(response)
+    end
+  end
+end
```
Comment (mnoack) [inline lib/quick_travel/price_quote.rb]: Interesting reading: https://robots.thoughtbot.com/no-newline-at-end-of-file

Comment (mnoack) [inline lib/quick_travel/price_quote.rb]: It seems the 2 key benefits are so when you cat a file your command prompt is on a new line, and so unix recognises it as a text file (again avoiding warnings)

**Comment by mnoack** [comment]
@AlessandroBerardi as discussed, can this test data be included in bootstrap.sql before merging this.

**Comment by alxberardi** [comment]
Agreed @mnoack , thanks!

File: lib/quick_travel/price_quote.rb (hunk 1 above)
Comment (alxberardi) [inline lib/quick_travel/price_quote.rb]: I'll fix this, thanks!

**Comment by alxberardi** [comment]
:warning: Needs https://bitbucket.org/team-sealink/quicktravel_test/pull-requests/17/bootstrap-db-update/diff :warning:

**Comment by alxberardi** [comment]
@mnoack Please see latest commits and https://bitbucket.org/team-sealink/quicktravel_test/pull-requests/18/bootstrap-db-update/diff

File: lib/quick_travel/price_quote.rb (hunk 1 above)
Comment (alxberardi) [inline lib/quick_travel/price_quote.rb]: Fixed

**Comment by msmirnoff** [comment]
👍

**Pull request merged**